RUFAS/input_manager.py

```diff
@@ -97,20 +106,29 @@ def _load_properties(self) -> None:
         Exception
             For any other unexpected errors during properties loading.
         """
-        info_map = {"class": self.__class__.__name__,
-                    "function": self._load_properties.__name__,
-                    }
+        info_map = {
+            "class": self.__class__.__name__,
+            "function": self._load_properties.__name__,
+        }
```

Comment:
```suggestion
        info_map = {"class": self.__class__.__name__, "function": self._load_properties.__name__}
```
This is an example of how Black is really good, but has a few buried rules that can lead to unexpected formatting choices. The above change isn't actually necessary, I just wanted to point this out to people.

Reply:
It's a good point. I think Black focuses on its predetermined definition of readability to the nth degree. So it's not a necessary change, but does make the same choice across the board so things look consistent. I think I've commented on how strange some of the choices look but maybe once the RUFAS team has been using Black for long enough these will all look like the norm to us.